__tests__/run.test.ts

```diff
@@ -321,7 +323,8 @@ test("deployment - deploy() - Annotate & label resources", async () => {
 
 test("deployment - deploy() - Annotate & label resources for a new workflow", async () => {
     process.env.GITHUB_WORKFLOW = '.github/workflows/NewWorkflow.yml';
-    let annotationKeyValStr = getWorkflowAnnotationKeyLabel(process.env.GITHUB_WORKFLOW) + '=' + getWorkflowAnnotationsJson('NA', '.github/workflows/NewWorkflow.yml');
+    let filepathConfigs = { manifestFilePaths :['manifests/deployment.yaml'], helmChartFilePaths : '', buildConfigs :{}} ;
+    let annotationKeyValStr = getWorkflowAnnotationKeyLabel(process.env.GITHUB_WORKFLOW) + '=' + getWorkflowAnnotationsJson('NA', '.github/workflows/NewWorkflow.yml', filepathConfigs);
     const KubernetesManifestUtilityMock = mocked(KubernetesManifestUtility, true);
```

Comment:
Add a new test case for your changes around dockerfile path and helm chart file paths.

Reply:
Will add it in next iteration, after the paths json structures are final